frontend/src/components/EncryptedChat.js

```diff
@@ -270,23 +271,11 @@ class Chat extends Component {
         <CardHeader
           sx={{ color: '#333333' }}
           avatar={
-            <Badge
-              variant='dot'
-              overlap='circular'
-              badgeContent=''
-              color={props.userConnected ? 'success' : 'error'}
-            >
-              <Avatar
-                className='flippedSmallAvatar'
-                alt={props.message.userNick}
-                src={
-                  window.location.origin +
-                  '/static/assets/avatars/' +
-                  props.message.userNick +
-                  '.png'
-                }
-              />
-            </Badge>
+            <RobotAvatar
+              avatarClass="phoneFlippedSmallAvatar"
```

Comment:
Not sure if intentional but `className="phoneFlippedSmallAvatar"` is the button Profile button for mobile. It's an avatar modificed to be more of a "button". You have used it here instead of `className='flippedSmallAvatar'`. Now chat avatars look like this in both desktop and mobile:
![chat](https://user-images.githubusercontent.com/90936742/191736866-5716707d-5a47-4cdc-8b0d-803d8a5c6db5.png)
They used to look like this:
![chat2](https://user-images.githubusercontent.com/90936742/191737322-02be779c-8bb4-4b4b-8440-ab8a24b5b2ad.png)

Reply:
Yeah that was a typo